python/felis/datamodel.py

```diff
@@ -462,3 +465,27 @@ def require_description(cls, rd: bool = True) -> None:
     def is_description_required(cls) -> bool:
         """Return whether a description is required for all objects."""
         return cls.ValidationConfig._require_description
+
+    @classmethod
+    def from_yaml_path(cls, yaml_path: str) -> Schema:
+        with open(yaml_path, "r") as yaml_file:
+            yaml_data = yaml.load(yaml_file, Loader=yaml.SafeLoader)
+        schema = cls.model_validate(yaml_data)
+        return schema
+
+    @classmethod
+    def from_yaml_file(cls, yaml_file: TextIOBase) -> Schema:
```

Comment:
I think `file` in this context is confusing since people first think you mean a file path and not a stream.

Reply:
see https://github.com/lsst/felis/pull/47#discussion_r1525442939